vmware/ngc/NGC-Plugin/src/test/java/org/opensds/vmware/ngc/service/impl/DatastoreTest.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * 
+ */
+
+package service.impl;
```

Comment:
Please change to "package org.opensds.vmware.ngc.service.impl;"

Reply:
Was my bad. Changed now. please check